build/checkstyle.xml

```
@@ -48,6 +48,9 @@
     <module name="EmptyStatement"/>
     <module name="SuperFinalize"/>
     <module name="EqualsHashCode"/>
+    <module name="EmptyCatchBlock">
+      <property name="exceptionVariableName" value="ignore|ignored"/>
```

It signals that the empty catch clause is intentional, rather than accidental.